Assignment/Assignment.Tests/PingProcessTests.cs

```diff
@@ -57,44 +58,42 @@ public void Run_CaptureStdOutput_Success()
     [TestMethod]
     public void RunTaskAsync_Success()
     {
-        // Do NOT use async/await in this test.
-        // Test Sut.RunTaskAsync("localhost");
+        Task<PingResult> testTask = Sut.RunTaskAsync("localhost");
+        AssertValidPingOutput(testTask.Result);
     }
 
     [TestMethod]
     public void RunAsync_UsingTaskReturn_Success()
     {
-        // Do NOT use async/await in this test.
-        PingResult result = default;
-        // Test Sut.RunAsync("localhost");
-        AssertValidPingOutput(result);
+        AssertValidPingOutput(Sut.RunAsync("localhost").Result);

```

Comment:
It's a bit more readable if you separate function calls into variables (like here, having RunAsync return as a PingResult object) so it's clear what you're trying to test with at first glance.

```suggestion
        PingResult result = Sut.RunAsync("localhost").Result);
        
        AssertValidPingOutput(result);

```

Reply:
thank you for the suggestion, it does provide more readability!